qa/workunits/rbd/rbd_mirror_ha.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/sh
```

Comment:
@trociny Just realized while running the tests that this needs an associate suite test case to execute it under teuthology